config/containers/logging/dual-logging.md

```diff
@@ -21,9 +21,16 @@ logs regardless of the configured logging driver or plugin. This capability,
 referred to as "dual logging", allows you to use `docker logs` to read container
 logs locally in a consistent format, regardless of the log driver used, because
 the engine is configured to log information to the “local” logging driver. Refer
-to [Configure the default logging driver](/config/containers/logging/configure)
-for additional information. 
+to [Configure the default logging driver](configure.md) for additional information. 
 
+Dual logging uses the [`local`](local.md) logging driver to act as cache for
+reading the latest logs of your containers. By default, the cache has log-file
+rotation enabled, and is limited to a maximum of 5 files of 20MB each (before
+compression) per container.
+
+Refer to the [configuration options](#configuration-options) section to customize
+these defaults, or to the [disable dual-logging](#disable-dual-logging) section
```

Comment:
Should this be `(#disable-the-dual-logging-cache)`?

Reply:
Good catch; l must've changed it after I wrote this; let me fix

Reply:
fixed 👍